base_konnector.js

```diff
@@ -41,6 +41,17 @@ module.exports = {
           process.exit(0)
         })
         .then(account => {
+          // get the folder path from the folder id and put it in cozyFields.folderPath
+          return new Promise((resolve, reject) => {
+            cozy.files.statById(cozyFields.folderPath, false)
```

Comment:
So maybe we may consider this issue the other way : why do we have an id instead of a path here ?